tests/integration/components/file-browser/component-test.ts

```diff
@@ -0,0 +1,85 @@
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { ModelInstance } from 'ember-cli-mirage';
+import { TestContext } from 'ember-intl/test-support';
+import { MirageRegistration } from 'ember-osf-web/mirage/factories/registration';
+import { click } from 'ember-osf-web/tests/helpers';
+import { setupRenderingTest } from 'ember-qunit';
+// import moment from 'moment';
+import { module, test } from 'qunit';
+import { FileItemKinds } from 'ember-osf-web/models/base-file-item';
+import FileProviderModel from 'ember-osf-web/models/file-provider';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import FileModel from 'ember-osf-web/models/file';
+
+interface FileBrowserTestContext extends TestContext {
+    mirageRegistration: ModelInstance<MirageRegistration>;
+    osfStorageProvider: FileProviderModel;
+    topLevelFiles: Array<ModelInstance<FileModel>>;
+    topLevelFolder: ModelInstance<FileModel>;
+    secondaryLevelFiles: Array<ModelInstance<FileModel>>;
+}
+
+module('Integration | Component | file-browser', hooks => {
+    setupRenderingTest(hooks);
+    setupMirage(hooks);
+    hooks.beforeEach(function(this: FileBrowserTestContext) {
+        this.store = this.owner.lookup('service:store');
+        this.mirageRegistration = server.create('registration');
```

Comment:
Because we have to way to query for a `FileProvider`, we need to access it through a relationship to `Registration`.